### DISCUSSION TITLE
GitHub Pages Can not access

### DISCUSSION THREAD

**Discussion by ChibaMai** [community/community#22693] [Other Feature Feedback, Questions, & Ideas]
<p>Hello, why can’t the connection generated after the current github build completes github pages be accessed?</p>

**Comment by otter-computer** [comment]
<p>I’m not quite sure what you mean. Are you having trouble loading a GitHub Pages site? If so, can you send a link to the page/repository you’re having trouble with?</p>

**Comment by ChibaMai** [comment]
<p>GitHub Pages cannot be accessed</p>
<p>Generate static access resources. I don’t know what happened. It seems that all GitHub Pages are not accessible anymore.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/0/007913557a2f5867db2dc9b909ff48bbe74631ed.jpeg" data-download-href="/uploads/short-url/4boZyOtnzifgKYN8KqPeddTAst.jpeg?dl=1" title="20200811125348" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/44936398/181072615-c668bf84-5e27-4030-ab71-4ab65f66c7ab.jpeg" alt="20200811125348" data-base62-sha1="4boZyOtnzifgKYN8KqPeddTAst" width="690" height="293" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/0/007913557a2f5867db2dc9b909ff48bbe74631ed_2_690x293.jpeg, https://aws1.discourse-cdn.com/github/original/2X/0/007913557a2f5867db2dc9b909ff48bbe74631ed.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/2X/0/007913557a2f5867db2dc9b909ff48bbe74631ed.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/0/007913557a2f5867db2dc9b909ff48bbe74631ed_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">20200811125348</span><span class="informations">956×406 89.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by ChibaMai** [comment]
<p>My GitHub username：Thomasvader<br>
project address：<a href="https://github.com/Thomasvader/YuXin" rel="nofollow noopener">https://github.com/Thomasvader/YuXin</a></p>
<p>Attach a picture</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/c/c9b505ed6f0fc66a289e1bbb98af1c8059aa6a9c.jpeg" data-download-href="/uploads/short-url/sMnOkRBntmxnXxfDRhF7Bo7smZK.jpeg?dl=1" title="20200811125615" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/44936398/181072616-37b3736a-8d0d-4ddb-89bb-447736267a8e.jpeg" alt="20200811125615" data-base62-sha1="sMnOkRBntmxnXxfDRhF7Bo7smZK" width="447" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/c/c9b505ed6f0fc66a289e1bbb98af1c8059aa6a9c_2_447x500.jpeg, https://aws1.discourse-cdn.com/github/original/2X/c/c9b505ed6f0fc66a289e1bbb98af1c8059aa6a9c.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/2X/c/c9b505ed6f0fc66a289e1bbb98af1c8059aa6a9c.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/c/c9b505ed6f0fc66a289e1bbb98af1c8059aa6a9c_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">20200811125615</span><span class="informations">600×670 83.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by otter-computer** [comment, marked as answer]
<p>Thanks for posting the link! I took a look at your site and it seems like it’s now loading the HTML content, but isn’t loading any CSS, JS, or other assets.</p>
<p>This looks like it’s happening because all of your asset links have <code>/yuxin/</code> prepended to their links. For example, looking at the link to your <code>main.css</code> file:</p>
<aside class="onebox githubblob">
  <header class="source">
      <a href="https://github.com/Thomasvader/YuXin/blob/3e47c77d20fd2dd6a9a6c82bb2b9e6ec8e5c91b4/index.html#L60" target="_blank" rel="noopener">github.com</a>
  </header>
  <article class="onebox-body">
    <h4><a href="https://github.com/Thomasvader/YuXin/blob/3e47c77d20fd2dd6a9a6c82bb2b9e6ec8e5c91b4/index.html#L60" target="_blank" rel="noopener">Thomasvader/YuXin/blob/3e47c77d20fd2dd6a9a6c82bb2b9e6ec8e5c91b4/index.html#L60</a></h4>
<pre class="onebox"><code class="lang-html"><ol class="start lines" start="50" style="counter-reset: li-counter 49 ;">
<li>  </li>
<li>
</li>
<li>&lt;link href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free/css/all.min.css" rel="stylesheet" type="text/css" /&gt;</li>
<li>
</li>
<li>
</li>
<li>
</li>
<li>  </li>
<li>
</li>
<li>&lt;link href="https://cdn.jsdelivr.net/npm/social-share.js@1.0.16/dist/css/share.min.css" rel="stylesheet" type="text/css" /&gt;</li>
<li>
</li>
<li class="selected">&lt;link href="/yuxin/css/main.css?v=0.0.68" rel="stylesheet" type="text/css" /&gt;</li>
<li>
</li>
<li>  &lt;link rel="apple-touch-icon" sizes="180x180" href="/yuxin/apple-touch-icon.png?v=0.0.68"&gt;</li>
<li>
</li>
<li>
</li>
<li>  &lt;link rel="icon" type="image/png" sizes="32x32" href="/yuxin/images/favicon.ico?v=0.0.68"&gt;</li>
<li>
</li>
<li>
</li>
<li>  &lt;link rel="mask-icon" href="/yuxin/safari-pinned-tab.svg?v=0.0.68" color="#fff"&gt;</li>
<li>
</li>
<li>
</li>
</ol></code></pre>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>GitHub Pages file servers are case sensitive so the case for these links needs to match the case or your repository name. As your repository is called <code>YuXin</code> any links that contain <code>yuxin</code> will lead to a 404.</p>
<p>Looking at how your site is built it seems like these links may be generated from a configuration file so you may want to update the <code>url</code> and <code>root</code> values here to match your site’s full URL and character case then redeploy your site:</p>
<aside class="onebox githubblob">
  <header class="source">
      <a href="https://github.com/Thomasvader/YuXin/blob/0637eeb3480714e192bd767a43f15643ed9c8848/_config.yml#L17" target="_blank" rel="noopener">github.com</a>
  </header>
  <article class="onebox-body">
    <h4><a href="https://github.com/Thomasvader/YuXin/blob/0637eeb3480714e192bd767a43f15643ed9c8848/_config.yml#L17" target="_blank" rel="noopener">Thomasvader/YuXin/blob/0637eeb3480714e192bd767a43f15643ed9c8848/_config.yml#L17</a></h4>
<pre class="onebox"><code class="lang-yml"><ol class="start lines" start="7" style="counter-reset: li-counter 6 ;">
<li>subtitle: Hi，语心</li>
<li># description: 前端萌新 | 开源 | 个人</li>
<li>keywords:</li>
<li># author: 语心</li>
<li>language: en</li>
<li>timezone: ''</li>
<li>
</li>
<li># URL</li>
<li>## If your site is put in a subdirectory, set url as 'http://yoursite.com/child' and root as '/child/'</li>
<li>url: https://ipvb.gitee.io/yuxin</li>
<li class="selected">root: /yuxin</li>
<li>permalink: :title/</li>
<li>permalink_defaults:</li>
<li>pretty_urls:</li>
<li>  # trailing_index: true # Set to false to remove trailing 'index.html' from permalinks</li>
<li>  # trailing_html: true # Set to false to remove trailing '.html' from permalinks</li>
<li>
</li>
<li># Directory</li>
<li>source_dir: source</li>
<li>public_dir: public</li>
<li>tag_dir: tags</li>
</ol></code></pre>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by ChibaMai** [comment]
<p>I changed the configuration file to reproduce the compilation, and it seems that the effect has not changed and it is not accessible.</p>

**Comment by otter-computer** [comment]
<p>Pages sites can take around 10 minutes to finish deploying after you make a push so if you visited your site too quickly after making that change then it’s likely you were still seeing the older version. Looks like it’s working now though!</p>
<p><a href="https://thomasvader.github.io/YuXin/" class="onebox" target="_blank">https://thomasvader.github.io/YuXin/</a></p>

**Comment by ChibaMai** [comment]
<p>Does Github not support GitHub Pages?</p>

**Comment by ChibaMai** [comment]
<p>I use Windows Ping to access <a href="http://thomasvader.github.io" rel="nofollow noopener">thomasvader.github.io</a> this URL returns 127.0.0.1.</p>
<p><img src="https://user-images.githubusercontent.com/44936398/181072620-ef224fec-f03e-4e54-b89c-c99ef1978b8a.jpeg" alt="20200819212002" data-base62-sha1="tT5cRVRuT20MtYD4HR3bCUNORsy" width="664" height="318"></p>

**Comment by otter-computer** [comment]
<p>From those results it looks like something is routing any requests to your site back to your local machine. Here’s what I get when I do the same:</p>
<pre><code class="lang-auto">% ping thomasvader.github.io
PING thomasvader.github.io (185.199.110.153): 56 data bytes
64 bytes from 185.199.110.153: icmp_seq=0 ttl=56 time=15.847 ms
64 bytes from 185.199.110.153: icmp_seq=1 ttl=56 time=22.277 ms
64 bytes from 185.199.110.153: icmp_seq=2 ttl=56 time=19.472 ms
64 bytes from 185.199.110.153: icmp_seq=3 ttl=56 time=22.716 ms
</code></pre>
<p>You may want to check if there’s something on your network that’s blocking access to <code>github.io</code> domains, or check you’ve not routed this domain to <code>127.0.0.1</code> in your hosts file.</p>

**Comment by ChibaMai** [comment]
<p>Sorry, China has banned DNS, which is really troublesome!</p>
